recipes/tests.rb

```diff
@@ -157,6 +159,7 @@
   bash 'test nvidia driver install' do
     cwd Chef::Config[:file_cache_path]
     code <<-TESTNVIDIA
+      set -e
```

Comment:
must be not put here, it's few lines later

Reply:
OK :+1: